contribs/gmf/src/directives/locationchooser.js

```diff
@@ -72,11 +74,6 @@ gmf.LocationchooserController = function($scope) {
    */
   this.locations = locations;
 
-  /**
-   * @type {gmfx.LocationchooserLocation}
-   * @export
-   */
-  this.selectedLocation = this.locations[0];
```

Comment:
Don't you want to use something like this here (assuming you use `bindToController`)?

```js
/**
 * @type {gmfx.LocationchooserLocation}
 * @export
 */
this.selectedLocation;

if (!goog.isDef(this.selectedLocation)) {
  this.selectedLocation = this.locations[0];
}
```